descriptors/ModuleDescriptor-template.json

```diff
@@ -1039,6 +1039,31 @@
         "inn-reach.marc-record-transformation.item.get"
       ]
     },
+    {
+      "permissionName" : "inn-reach.central-servers.visible-patron-field-configuration.item.get",
```

Comment:
Apart from the permissions there should be endpoints defined also, I don't see them